src/utils/feature-flags.ts

```diff
@@ -13,7 +13,7 @@ const defaultFlagSet: LDFlagSet = {
   'sentry-enable': false, // by default, no sentry logs
   'show-alert-phone-numbers-firm': false,
   'supported-business-summary-entities': [],
-  'supported-correction-entities': [],
+  'supported-correction-entities': ['CP'],
```

Comment:
This is the local default, maybe better to turn this on in LaunchDarkly?

Reply:
Already on

Reply:
The default should be "feature is disabled" otherwise the code will fall back to enabled if LaunchDarkly can't be reached (which you can do from dev tools).

Revert this, please.

Reply:
PS - For local testing, your env file should specify the Dev LD key.

Reply:
In DEV, LD already has CP included in the feature flag:
![correction entities](https://github.com/bcgov/business-filings-ui/assets/122301442/2691a100-1324-42f8-af8c-0826df622263)

Reply:
Jia, please let the author of a conversation resolve it (or someone else if they post a comment saying so). This allows reviewers to check only their outstanding conversations... otherwise we have to review everything again and look at resolved conversations to see if the requested changes are OK. Thanks! (And no worries about this time.)